packages/runtime/src/prop-controllers/descriptors.ts

```diff
@@ -215,6 +216,7 @@ type BackgroundImageV1 = {
   repeat?: BackgroundImageRepeat
   opacity?: number
   parallax?: number
+  priority?: boolean
```

Comment:
See my other comment on adding `priority` to `BackgroundImageV0`

Reply:
Same response.